code/datums/status_effects/neutral.dm

```diff
@@ -40,12 +40,9 @@
 
 /datum/status_effect/high_five
 	id = "high_five"
-	duration = 40
+	duration = 5 SECONDS
 	alert_type = null
 
-/datum/status_effect/high_five/on_remove()
-	owner.visible_message("[owner] was left hanging....")
```

Comment:
Oh, okay! I removed this because it left me with the message regardless of how it got removed, wasn't aware that this was a known bug before my changes. I wasn't too well acquainted with status effects (though I'm absolutely sure you're well acquainted whether you like it or not :^) ) so I'll see about fiddling with this!